pkg/util/test/util.go

```diff
@@ -728,6 +735,7 @@ func defaultPxNodeAffinityRules() *v1.NodeAffinity {
 func validatePortworxNodes(cluster *corev1.StorageCluster, expectedNodes int) error {
 	conn, err := getSdkConnection(cluster)
 	if err != nil {
+		// CHECKME -- shouldn't we return err ?
```

Comment:
There is probably a lower version of px that does not have the support. Even in that case, the error should be checked before return nil